src/platforms/javascript/guides/nextjs/manual-setup/sentry-properties.mdx

```diff
@@ -0,0 +1,33 @@
+---
+title: Sentry properties
+sidebar_order: 2
+description: "Learn other ways to set the Sentry properties for your application."
+---
+
+If you can’t (or prefer not to) run the [setup step](/platforms/javascript/guides/nextjs/#setup), you can follow the instructions below to configure your application.
+
+## A properties file
+
+You can create a `sentry.properties` file in the base directory of your project, as in the [setup step](/platforms/javascript/guides/nextjs/#setup):
+
+```properties {filename:sentry.properties}
+defaults.url=https://sentry.io/
+defaults.org=___ORG_SLUG___
+defaults.project=___PROJECT_SLUG___
+auth.token=___TOKEN___
+# cli.executable=../path/to/bin/sentry-cli
+```
+
+## Using environment variables
+
+Using environment variables allows to have a more exhaustive control over the Sentry parameters the webpack plugin needs, and even to get rid of the properties file.
+If you choose to combine both, the plugin will choose environment variables over the parameters in the properties file.
+
+| Property name      | Environment variable |
+| ------------------ | -------------------- |
+| `defaults.url`     | `SENTRY_URL`         |
+| `defaults.org`     | `SENTRY_ORG`         |
+| `defaults.project` | `SENTRY_PROJECT`     |
+| `auth.token`       | `SENTRY_AUTH_TOKEN`  |
+
+You can also set the Sentry DSN using the environment variable `SENTRY_DSN` or `NEXT_PUBLIC_SENTRY_DSN`, instead of having it in the `next.config.js`.
```

Comment:
```suggestion
You can also set the Sentry DSN using the environment variable `SENTRY_DSN` or `NEXT_PUBLIC_SENTRY_DSN` instead of having it in the `next.config.js`.
```